tests/Batch/output/all_openvino_models_i7-1370PRE_CPU_iGPU/bert-large-uncased-whole-word-masking-squad-int8-0001_08_25_2023-16_12_19_log.txt

```diff
@@ -0,0 +1,312 @@
+Error benchmarking model
```

Comment:
`Output` folder contains too many large files that are not really needed for the tool to work. 
You can leave a small example or describe everything in the `README`.
Could you please remove unnecessary logs from the PR and leave only needed files (with the source code)?

Reply:
`Batch/*.ipynb` analysis files are quite representative. I believe it should be okay to keep them.
If OpenVINO maintainers do not mind.